manifests/0000_50_cluster-monitoring-operator_04-config.yaml

```diff
@@ -147,6 +153,11 @@ data:
     #
     # owners: (@openshift/openshift-team-monitoring, @smarterclayton)
     #
+    # up identifies hostname and internal IP of clusters
+    #
+    # consumers: (@openshift/openshift-team-cluster-manager)
+    - '{__name__="up", job="kubelet"}'
```

Comment:
We have `count:up0` and `count:up1` already and we don't want to send the raw `up` metric because it has a too high cardinality.

Reply:
removed